app/resources/forms.js

```diff
@@ -284,6 +284,22 @@ function validate_fields() {
   }
   hide($('mandatory_field_missing'));
 
+  //Validate email-id
+  var email_field = $('author_email');
+  var email_id = $('author_email').value;
+  if (email_id !== "")
+  {
```

Comment:
please follow the formatting convention in the file.

e.g.

if (email_id !== '') {
  ...
}